.travis.yml

```diff
@@ -1,17 +1,19 @@
+---
 language: ruby
 rvm:
-- 2.5.7
-- 2.6.5
+- 2.5.8
+- 2.6.6
 arch:
 - amd64
 - ppc64le
 cache:
   bundler: true
   directories:
-  - $HOME/.cache/go-build
-  - $HOME/gopath/pkg/mod
+  - "$HOME/.cache/go-build"
+  - "$HOME/gopath/pkg/mod"
 before_script:
 - eval "$(gimme 1.14)"
-- curl -L https://github.com/operator-framework/operator-sdk/releases/download/v0.15.1/operator-sdk-v0.15.1-$(uname -m)-linux-gnu -o /home/travis/bin/operator-sdk && chmod +x /home/travis/bin/operator-sdk
+- curl -L https://github.com/operator-framework/operator-sdk/releases/download/v0.15.1/operator-sdk-v0.15.1-$(uname
+  -m)-linux-gnu -o /home/travis/bin/operator-sdk && chmod +x /home/travis/bin/operator-sdk
```

Comment:
Is this supposed to be on a new line?

Reply:
No, let me fix it.  Yaml dump must not like unquoted long lines.

Reply:
ok, fixed, @bdunne.  I kept the quotes added by YAML.dump since they seem pretty benign.